rllib/policy/sample_batch.py

```diff
@@ -960,31 +960,28 @@ def _get_slice_indices(self, slice_size):
                 i += slice_size
         return data_slices, data_slices_states
 
-    # TODO: deprecate
-    @property
-    def data(self):
-        deprecation_warning(
-            old="SampleBatch.data[..]", new="SampleBatch[..]", error=True)
-        return self
-
-    # TODO: (sven) Experimental method.
-    def get_single_step_input_dict(self, view_requirements, index="last"):
+    @ExperimentalAPI
+    def get_single_step_input_dict(
+            self,
+            view_requirements: ViewRequirementsDict,
+            index: Union[str, int] = "last",
```

Comment:
it seems like index can be a tuple too. maybe update the annotation and doc string.

Reply:
Great catch!

Reply:
I took out the Tuple option. Never used and doesn't seem to make any sense here.
We only ever use "last" or -1 (in the MARWIL algo).

Reply:
👌